test/viewport.test.js

```diff
@@ -33,6 +33,14 @@ describe('o-viewport utils', function () {
 		clock.tick(1);
 		expect(callback.callCount).to.be(1);
 
+		throttled();
```

Comment:
Oh, what I meant is that the throttled calls should go one after the other, and then only be executed after those initial 100ms